src/devices/Bmxx80/ReadResult/Bme280ReadResult.cs

```diff
@@ -1,6 +1,5 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
```

Comment:
I'm ok with this being in this PR only because this technically speaking doesn't change the product